.github/workflows/test.yml

```diff
@@ -0,0 +1,88 @@
+name: Test
+
+on:
+  pull_request:
+    branches: [main]
+    paths: '**.py'
```

Comment:
Shall we add `*.ipynb` to see if a change in a notebook that will break the `example_notebooks` job will trigger the CI and fail as expected?

Reply:
How about putting that into a separate action? The tests will only need to be run with package changes. The notebooks with package **or** notebooks themselve

Reply:
I think it's a good idea to separate them out. We would want to trigger the `example_noteboooks` on any change to an `.ipynb` or `.py`.

Reply:
Lets give the separation a try